openhands/sdk/conversation/conversation.py

```diff
@@ -45,6 +46,7 @@ def __init__(
         callbacks: list[ConversationCallbackType] | None = None,
         max_iteration_per_run: int = 500,
         visualize: bool = True,
+        stuck_detection: bool = False,
```

Comment:
Should default to True

```suggestion
        stuck_detection: bool = True,
```